src/target/riscv/riscv.c

```diff
@@ -5847,6 +5848,52 @@ bool reg_is_initialized(const struct reg *reg)
 	return true;
 }
 
+static struct target *reg_get_target(const struct reg *reg)
+{
+	assert(reg_is_initialized(reg));
+	return ((const riscv_reg_info_t *)reg->arch_info)->target;
+}
+
+static int riscv011_reg_get(struct reg *reg)
+{
+	struct target * const target = reg_get_target(reg);
+	RISCV_INFO(info);
+	riscv_reg_t value;
+	const int result = info->get_register(target, &value, reg->number);
+	if (result != ERROR_OK)
+		return result;
+	buf_set_u64(reg->value, 0, reg->size, value);
+	return ERROR_OK;
+}
+
+static int riscv011_reg_set(struct reg *reg, uint8_t *buf)
+{
+	const riscv_reg_t value = buf_get_u64(buf, 0, reg->size);
+	struct target * const target = reg_get_target(reg);
+	RISCV_INFO(info);
```

Comment:
Same as above - please insert an assertion here:

```suggestion
	RISCV_INFO(info);
	assert(info->dtm_version == DTM_DTMCS_VERSION_0_11);	
```

Reply:
Addressed